src/lib/commands/setup-draw-command.ts

```diff
@@ -71,6 +72,13 @@ export function setupDrawCommand(
       color: [0, 0, 0, 0]
     },
 
+    viewport: () => ({
```

Comment:
Adding this fixed rendering issues post-resize. Without it, WebGL captures the viewport on first render and never changes it. The effect is big blank areas if the canvas has gotten larger.